tests/core/test_memory.py

```diff
@@ -163,16 +215,16 @@ def test_rnn_summary_shapes(mode):
         [b, t, i],  # rnn
         [b, t, h],  # linear
     ]
-    assert summary.out_sizes == [
-        [[b, t, h], [[1, b, h], [1, b, h]]],    # rnn
-        [b, t, o]                               # linear
-    ]
+    assert summary.out_sizes == [[[b, t, h], [[1, b, h], [1, b, h]]], [b, t, o]]  # rnn  # linear
 
 
-@pytest.mark.parametrize(['mode'], [
-    pytest.param(ModelSummary.MODE_FULL),
-    pytest.param(ModelSummary.MODE_TOP),
-])
+@pytest.mark.parametrize(
+    ["mode"],
+    [
+        pytest.param(ModelSummary.MODE_FULL),
+        pytest.param(ModelSummary.MODE_TOP),
+    ],
+)
```

Comment:
```suggestion
@pytest.mark.parametrize(["mode"], ModelSummary.MODES)
```

Reply:
It works if you do instead:

`@pytest.mark.parametrize("mode", ModelSummary.MODES)`

Same for all other ocurrences.

Sorry for the confusion!